docs/recipes/pantheon.md

```diff
@@ -45,7 +45,7 @@ git clone https://github.com/kalabox/lando.git
 cd lando/examples/pantheon
 
 # Clone drops-7 into a nested docroot
-git clone git@github.com:pantheon-systems/drops-7.git web
+git clone https://github.com/pantheon-systems/drops-7.git web
```

Comment:
My guess is the reason this change is necessary is because your Pantheon account doesn't have the ssh pubkey yet?  I believe Terminus has commands now that let you manage ssh keys.  Maybe add a key and it will work?